service/framework.go

```diff
@@ -232,9 +305,10 @@ func newCRDFramework(config Config) (*framework.Framework, error) {
 	// We provide a map of resource lists keyed by the version bundle version
 	// to the resource router.
 	versionedResources := map[string][]framework.Resource{
-		"1.0.0": resources,
-		"0.1.0": resources,
-		"":      resources,
+		"2.0.0": resourcesV3,
```

Comment:
Asking again, why is this `2.0.0` now? I thought one feature is added. There is no breaking change, so `1.1.0` makes sense to me.

Reply:
rebase problem probably